### PR TITLE
Add Login button to pages without one : RE

### PR DESCRIPTION
**Pull request by ShailRT**
[//]: # (Thank you for uploading a PR to the Wikipedia Library!)

## Description
Login button added to the pages that can be accessed without logging in and return users back to the page they were previously on. 
(https://www.mediawiki.org/wiki/Gerrit/Commit_message_guidelines)

## Phabricator Ticket
https://phabricator.wikimedia.org/T299638

## How Has This Been Tested?
it was tested by going to a page that is accessissible without logging in and observing the changes there
eg link - http://localhost/password/reset/

## Screenshots of your changes (if appropriate):
https://drive.google.com/drive/u/0/folders/1mV6_KTTXSvVx1NLyzhDl8mkNNAIXO6iI

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Minor change (fix a typo, add a translation tag, add section to README, etc.)

### CODE REVIEW

File: TWLight/templates/header_partial_b4.html [hunk 1]
```diff
@@ -104,6 +104,12 @@
                 {% trans "Log out" %}
               </a>
             </li>
+          {% elif not user.is_authenticated %}
+            <li role="presentation" class="mobile-menu align-middle nav-item">
+              <a class="btn twl-btn" href="{% url 'auth_login' %}?next={{request.path}}">
```
Comment (suecarmol) [inline TWLight/templates/header_partial_b4.html:109]: Instead of using the Django `auth_login` default view, can you please change it to `oauth_login`?

**Review by suecarmol — CHANGES_REQUESTED** [review]
Thank you for working on this new feature! You need to make some changes to your PR before I can merge it.

File: TWLight/templates/header_partial_b4.html (hunk 1 above)
Comment (ShailRT) [reply TWLight/templates/header_partial_b4.html:109]: is oauth login pre-created or do i have to create it?

Comment (Samwalton9) [reply TWLight/templates/header_partial_b4.html:109]: It already exists, you can see it being used here, for example: https://github.com/WikipediaLibrary/TWLight/blob/master/TWLight/templates/login_partial.html#L64

**Comment by suecarmol** [comment]
Closing this because we already merged #794. Thank you for working on this :)

**Pull request closed without merging**